contracts/handlers/PermissionlessGenericHandler.sol

```diff
@@ -61,7 +58,7 @@ contract GenericHandlerV1 is IGenericHandler {
           executionDataDepositor:       bytes    bytes  36 + len(executeFuncSignature) + len(executeContractAddress)                                -  36 + len(executeFuncSignature) + len(executeContractAddress) + len(executionDataDepositor)
           executionData:                bytes    bytes  36 + len(executeFuncSignature) + len(executeContractAddress) + len(executionDataDepositor)  -  END
      */
-    function deposit(bytes32 resourceID, address depositor, bytes calldata data) external returns (bytes memory) {
+    function deposit(bytes32 resourceID, address depositor, bytes calldata data) external onlyBridge returns (bytes memory) {
```

Comment:
I guess it wasn't onlyBridge because it doesn't do anything except of checks? Also should be made `view`.

Reply:
Yep, but I realized it would be better to add `onlyBridge` so it's consistent across all handlers.